x-pack/legacy/plugins/actions/server/init.ts

```diff
@@ -55,6 +64,12 @@ export function init(server: Legacy.Server) {
     getServices,
     taskManager: taskManager!,
     encryptedSavedObjectsPlugin: server.plugins.encrypted_saved_objects!,
+    getBasePath(...args) {
+      return spaces.isEnabled ? spaces.getBasePath(...args) : '';
```

Comment:
`spaces.getBasePath()` currently includes the server base path (e.g. `/serverBasePath/s/foo`), so when spaces is disabled, you'll likely want to return `server.config().get('server.basePath')` instead of `''`